install/kubernetes/cilium/charts/hubble-relay/README.md

```diff
@@ -0,0 +1,33 @@
+# hubble-relay
+
+![Version: 1.7.9](https://img.shields.io/badge/Version-1.7.9-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.7.9](https://img.shields.io/badge/AppVersion-1.7.9-informational?style=flat-square)
+
+A Helm chart for Kubernetes
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| debug | bool | `false` |  |
+| dialTimeout | string | `nil` |  |
+| enabled | bool | `false` |  |
+| global.hubble.socketPath | string | `"/var/run/cilium/hubble.sock"` |  |
+| image.pullPolicy | string | `"IfNotPresent"` |  |
+| image.repository | string | `"quay.io/isovalent/hubble-relay"` |  |
+| image.tag | string | `"latest"` |  |
+| imagePullSecrets | string | `nil` |  |
+| listenHost | string | `""` |  |
+| listenPort | string | `"4245"` |  |
+| numReplicas | int | `1` |  |
+| peerService.image.pullPolicy | string | `"IfNotPresent"` |  |
+| peerService.image.repository | string | `"quay.io/isovalent/hubble-k8s-peer-service"` |  |
+| peerService.image.tag | string | `"latest"` |  |
+| podSecurityPolicy.enabled | bool | `false` |  |
+| resources | object | `{}` |  |
+| retryTimeout | string | `nil` |  |
+| serviceAccount.annotations | string | `nil` |  |
+| serviceAccount.create | bool | `true` |  |
+| servicePort | int | `80` |  |
+| socketPath | string | `"/var/run/cilium/hubble.sock"` |  |
+| sortBufferDrainTimeout | string | `nil` |  |
+| sortBufferLenMax | string | `nil` |  |
```

Comment:
Are these auto-generated? What's the plan to keep them in sync with the charts?

Reply:
these are auto-generated, I'll also add the makefile commands to do this and document how-to